core/frontend/src/types/autopilot/parameter-rover-enums.ts

```diff
@@ -0,0 +1,80 @@
+export enum FRAME_TYPE {
+  UNDEFINED = 0,
+  OMNI3 = 1,
+  OMNIX = 2,
+  OMNIPLUS = 3,
+}
+
+export enum FRAME_CLASS {
+  UNDEFINED = 0,
+  ROVER = 1,
+  BOAT = 2,
+  BALANCEBOT = 3,
+}
+
+export enum SERVO_FUNCTION {
+    // TODO: double-check
+    DISABLED = 0,
+    RCPASSTHRU = 1,
+    FLAP = 2,
+    FLAP_AUTO = 3,
+    AILERON = 4,
+    MOUNT_PAN = 6,
+    MOUNT_TILT = 7,
+    MOUNT_ROLL = 8,
+    MOUNT_OPEN = 9,
+    CAMERA_TRIGGER = 10,
+    RELEASE = 11,
+    MOUNT2_PAN = 12,
+    MOUNT2_TILT = 13,
+    MOUNT2_ROLL = 14,
+    MOUNT2_OPEN = 15,
+    DIFFERENTIALSPOILER1 = 16,
+    DIFFERENTIALSPOILER2 = 17,
+    AILERONWITHINPUT = 18,
+    ELEVATOR = 19,
+    ELEVATORWITHINPUT = 20,
+    RUDDER = 21,
+    FLAPERON1 = 24,
+    FLAPERON2 = 25,
+    GROUNDSTEERING = 26,
+    PARACHUTE = 27,
+    EPM = 28,
+    LANDINGGEAR = 29,
+    ENGINERUNENABLE = 30,
+    HELIRSC = 31,
+    HELITAILRSC = 32,
+    MOTOR1 = 33,
+    MOTOR2 = 34,
+    MOTOR3 = 35,
+    MOTOR4 = 36,
+    MOTOR5 = 37,
+    MOTOR6 = 38,
+    MOTOR7 = 39,
+    MOTOR8 = 40,
+    MOTORTILT = 41,
+    RCIN1 = 51,
+    RCIN2 = 52,
+    RCIN3 = 53,
+    RCIN4 = 54,
+    RCIN5 = 55,
+    RCIN6 = 56,
+    RCIN7 = 57,
+    RCIN8 = 58,
+    RCIN9 = 59,
+    RCIN10 = 60,
+    RCIN11 = 61,
+    RCIN12 = 62,
+    RCIN13 = 63,
+    RCIN14 = 64,
+    RCIN15 = 65,
+    RCIN16 = 66,
+    IGNITION = 67,
+    CHOKE = 68,
+    STARTER = 69,
+    THROTTLE = 70,
+    TRACKERYAW = 71,
+    TRACKERPITCH = 72,
+    THROTTLELEFT = 73,
+    THROTTLERIGHT = 74,
+}
```

Comment:
```suggestion
export enum SERVO_FUNCTION {
  GPIO = -1,
  DISABLED = 0,
  RCPASSTHRU = 1,
  FLAP = 2,
  FLAPAUTO = 3,
  AILERON = 4,
  MOUNT1YAW = 6,
  MOUNT1PITCH = 7,
  MOUNT1ROLL = 8,
  MOUNT1RETRACT = 9,
  CAMERATRIGGER = 10,
  MOUNT2YAW = 12,
  MOUNT2PITCH = 13,
  MOUNT2ROLL = 14,
  MOUNT2RETRACT = 15,
  DIFFERENTIALSPOILERLEFT1 = 16,
  DIFFERENTIALSPOILERRIGHT1 = 17,
  ELEVATOR = 19,
  RUDDER = 21,
  SPRAYERPUMP = 22,
  SPRAYERSPINNER = 23,
  FLAPERONLEFT = 24,
  FLAPERONRIGHT = 25,
  GROUNDSTEERING = 26,
  PARACHUTE = 27,
  GRIPPER = 28,
  LANDINGGEAR = 29,
  ENGINERUNENABLE = 30,
  HELIRSC = 31,
  HELITAILRSC = 32,
  MOTOR1 = 33,
  MOTOR2 = 34,
  MOTOR3 = 35,
  MOTOR4 = 36,
  MOTOR5 = 37,
  MOTOR6 = 38,
  MOTOR7 = 39,
  MOTOR8 = 40,
  TILTMOTORSFRONT = 41,
  TILTMOTORSREAR = 45,
  TILTMOTORREARLEFT = 46,
  TILTMOTORREARRIGHT = 47,
  RCIN1 = 51,
  RCIN2 = 52,
  RCIN3 = 53,
  RCIN4 = 54,
  RCIN5 = 55,
  RCIN6 = 56,
  RCIN7 = 57,
  RCIN8 = 58,
  RCIN9 = 59,
  RCIN10 = 60,
  RCIN11 = 61,
  RCIN12 = 62,
  RCIN13 = 63,
  RCIN14 = 64,
  RCIN15 = 65,
  RCIN16 = 66,
  IGNITION = 67,
  STARTER = 69,
  THROTTLE = 70,
  TRACKERYAW = 71,
  TRACKERPITCH = 72,
  THROTTLELEFT = 73,
  THROTTLERIGHT = 74,
  TILTMOTORFRONTLEFT = 75,
  TILTMOTORFRONTRIGHT = 76,
  ELEVONLEFT = 77,
  ELEVONRIGHT = 78,
  VTAILLEFT = 79,
  VTAILRIGHT = 80,
  BOOSTTHROTTLE = 81,
  MOTOR9 = 82,
  MOTOR10 = 83,
  MOTOR11 = 84,
  MOTOR12 = 85,
  DIFFERENTIALSPOILERLEFT2 = 86,
  DIFFERENTIALSPOILERRIGHT2 = 87,
  WINCH = 88,
  MAINSAIL = 89,
  CAMERAISO = 90,
  CAMERAAPERTURE = 91,
  CAMERAFOCUS = 92,
  CAMERASHUTTERSPEED = 93,
  SCRIPT1 = 94,
  SCRIPT2 = 95,
  SCRIPT3 = 96,
  SCRIPT4 = 97,
  SCRIPT5 = 98,
  SCRIPT6 = 99,
  SCRIPT7 = 100,
  SCRIPT8 = 101,
  SCRIPT9 = 102,
  SCRIPT10 = 103,
  SCRIPT11 = 104,
  SCRIPT12 = 105,
  SCRIPT13 = 106,
  SCRIPT14 = 107,
  SCRIPT15 = 108,
  SCRIPT16 = 109,
  NEOPIXEL1 = 120,
  NEOPIXEL2 = 121,
  NEOPIXEL3 = 122,
  NEOPIXEL4 = 123,
  RATEROLL = 124,
  RATEPITCH = 125,
  RATETHRUST = 126,
  RATEYAW = 127,
  WINGSAILELEVATOR = 128,
  PROFILED1 = 129,
  PROFILED2 = 130,
  PROFILED3 = 131,
  PROFILEDCLOCK = 132,
  WINCHCLUTCH = 133,
  SERVONMIN = 134,
  SERVONTRIM = 135,
  SERVONMAX = 136,
  SAILMASTROTATION = 137,
  ALARM = 138,
  ALARMINVERTED = 139,
  RCIN1SCALED = 140,
  RCIN2SCALED = 141,
  RCIN3SCALED = 142,
  RCIN4SCALED = 143,
  RCIN5SCALED = 144,
  RCIN6SCALED = 145,
  RCIN7SCALED = 146,
  RCIN8SCALED = 147,
  RCIN9SCALED = 148,
  RCIN10SCALED = 149,
  RCIN11SCALED = 150,
  RCIN12SCALED = 151,
  RCIN13SCALED = 152,
  RCIN14SCALED = 153,
  RCIN15SCALED = 154,
  RCIN16SCALED = 155,
}
```

Reply:
done

Reply:
@Williangalvani it appears that the full table was not updated

Reply:
I hadn't realizes you added the new stuff. nice. updated.